scripts/sct_deepseg_lesion.py

```diff
@@ -47,7 +47,7 @@ def get_parser():
                       example="t1.nii.gz")
     parser.add_option(name="-c",
                       type_value="multiple_choice",
-                      description="type of image contrast.",
+                      description="type of image contrast. For 't2_like' contrast images, a specific model for images with axial orientation is available: 't2_ax'.",
```

Comment:
I would be more exhaustive in the description. E.g.: "type of image contrast. t2: T2w scan with isotropic resolution. t2_ax: T2w scan with axial orientation and thick slices. t2s: T2*w scan with axial orientation and thick slices."